sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,11 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (16336, 51966, 51971, 52010, 54089);
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(16336, 'spell_stratholme_haunting_phantoms'),
+(51966, 'spell_scourge_disguise'),
+(51971, 'spell_scourge_disguise_instability'),
+(52010, 'spell_scourge_disguise_expiring'),
+(54089, 'spell_drop_disguise');
+
+
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger` = 54089;
```

Comment:
suggestion: move this line on top and delete at least 2, maybe all 3 blank lines.